README.md

```diff
@@ -111,43 +111,6 @@ vt.assert_polygons(polygons_expected=polygons)
 Caveats: This repo likely misses edge cases of the many ways matplotlib plots can be created.
 Please feel free to submit bugs!
 
-
-## Active Contributors
-
-- Leah Wasser
-
-## Dev Setup (to be moved to contributing)
-
-setup the matplotcheck envt
-
-```
-conda env create -f environment.yml
-conda activate matplotcheck-dev
-```
-
-Then setup all of the development requirements.
-
-```
-pip install -e .
-pip install -r dev-requirements.txt
-pre-commit install
-```
-
-## Contributors
```

Comment:
This part stays in the readme but let's fix ryan's name. i see this was my mistake!! can you help me fix it?